dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -240,6 +247,30 @@ def writeErrors(self, failures, interfaces, job_id, short_colnames, writer, warn
             error_list.recordRowError(job_id,self.filename,field_name,error,row_number,original_rule_label,severity_id=severityId)
         return fatal_error_found
 
+    def writeToFlex(self, flex_cols, job_id, submission_id, file_type):
+        """ Write this record to the staging tables
+
+        Args:
+            flex_cols: Record to be written
+            job_id: ID of current job
+            submission_id: ID of current submission
+            file_type: Type of file for current job
+
+        Returns:
+            Boolean indicating whether to skip current row
+        """
+        sess = GlobalDB.db().session
+        try:
+            flex_cols["job_id"] = job_id
+            flex_cols["submission_id"] = submission_id
+
+            sess.add(FlexField(**flex_cols))
+            sess.commit()
+        except ResponseException as e:
```

Comment:
I don't think the above would raise a ResponseException -- is there some sqlalchemy exception you want to catch instead?